src/core/execute.c

```diff
@@ -1716,16 +1716,23 @@ static int exec_child(
                                 *exit_status = EXIT_SMACK_PROCESS_LABEL;
                                 return r;
                         }
-                }
 #ifdef SMACK_DEFAULT_PROCESS_LABEL
-                else {
-                        r = mac_smack_apply_pid(0, SMACK_DEFAULT_PROCESS_LABEL);
+                } else {
```

Comment:
Please put the the `}` *outside* of the `#ifdef` to keep things more obvious (like it was before). We already have two nested levels, and it's already too convoluted to quickly parse all possible combinations.